tests/unittests/MLTest.cpp

```diff
@@ -205,6 +207,79 @@ TEST_P(MLTest, learnXor) {
   }
 }
 
+/// Learn the logarithmic function
+TEST_P(MLTest, learnLog) {
+  unsigned numInputs = 500;
+  unsigned batchSize = 10;
+  EE_.getConfig().learningRate = 0.05;
+  EE_.getConfig().batchSize = batchSize;
+
+  auto &mod = EE_.getModule();
+  Function *F = mod.createFunction("main");
+  F->setName("learnLog");
+
+  auto *A =
+      mod.createVariable(ElemKind::FloatTy, {batchSize, 1}, "A", VisibilityKind::Public,
+                         Variable::TrainKind::None);
+  auto *Ex =
+      mod.createVariable(ElemKind::FloatTy, {batchSize, 1}, "Ex", VisibilityKind::Public,
+                         Variable::TrainKind::None);
+
+  Node *O = F->createFullyConnected("fc1", A, 5);
+  O = F->createTanh("tanh1", O);
+  O = F->createFullyConnected("fc2", O, 6);
+  O = F->createTanh("tanh2", O);
+  O = F->createFullyConnected("fc3", O, 1);
+  O = F->createTanh("tanh3", O);
+  O = F->createRegression("reg", O, Ex);
+  auto *result = F->createSave("ret", O);
+
+  //Set the training set 
```

Comment:
Space after the slashes. Period at the end.

Reply:
Done